src/platform/unix.rs

```diff
@@ -32,6 +32,9 @@ pub use crate::platform_impl::x11;
 #[cfg(feature = "x11")]
 pub use crate::platform_impl::{x11::util::WindowType as XWindowType, XNotSupported};
 
+#[cfg(feature = "wayland")]
+pub use sctk_adwaita::FrameConfig;
```

Comment:
I don't like pub reexports of dependencies.

I can see that you're using it just for `Dark` and `Light` theme variants, could you use https://github.com/rust-windowing/winit/blob/ce890c34551d9fb542f10dcb644d22d382e0c921/src/window.rs#L1100 instead? I think there're also methods to change theme on the `Window` already, they're just not exposed on Wayland/X11.

Reply:
This was done to allow custom themes, as Dark/Light are just two default ones, but nothing stops you from defining your own colors instead of calling dark()/light(), but sure I can just limit it to those two, it will simplify the API.

Reply:
Maybe I just read it in the wrong way. We had in the past structure to initialize Wayland csd stuff, could probably bring it back in some form, I just don't like reexporting, since it could create unnecessary breaking changes downstream.